framework/contracts/account/manager/src/commands.rs

```diff
@@ -224,6 +225,10 @@ pub fn create_subaccount(
             name,
             description,
             link,
+            install_modules,
+            // TODO?
+            base_asset: None,
```

Comment:
This is fixed in https://github.com/AbstractSDK/abstract/pull/45

Reply:
Yep will fix it after it gets merget into main